server/src/main/java/io/crate/analyze/ExplainAnalyzedStatement.java

```diff
@@ -44,25 +46,44 @@ public class ExplainAnalyzedStatement implements AnalyzedStatement, AnalyzedRela
     private final List<Symbol> outputs;
     private final RelationName relationName;
     private final boolean showCosts;
-    private static final String COLUMN_NAME = "QUERY PLAN";
+    private final boolean verbose;
+    private static final String PLAN_COLUMN_NAME = "QUERY PLAN";
+    private static final String STEP_COLUMN_NAME = "STEP";
 
-    ExplainAnalyzedStatement(AnalyzedStatement statement, @Nullable ProfilingContext context, boolean showCosts) {
+    ExplainAnalyzedStatement(AnalyzedStatement statement,
+                             @Nullable ProfilingContext context,
+                             boolean showCosts,
+                             boolean verbose) {
```

Comment:
Might be worth introducing a enumSet for the explain options. Something like `foo(ExplainOption.COSTS, ExplainOption.VERBOSE, ..)` is clearer when reading than `foo(true, false)`

(But also fine to keep as is for now)

Reply:
Done.